docs/src/use/configure/ignore.md

```diff
@@ -62,6 +62,61 @@ export default [
 ];
 ```
 
+## Global Ignores
+
+::: tip
+Global Ignores is what you need if you want to define files to ignore in a single place and avoid to add the same `ignores` prop to every configuration objects.
+:::
+
+In eslint, there is concept of `global ignores`, to better explain it let's see first how config object works in eslint.
+
+Eslint configuration file is intended to be used like this: export a single array, that contains multiple [configuration objects](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-objects), each one being an item of the array.  
+
+It's important to note that each configuration object, it's independent from other ones. Because of that, the `ignores` prop defined in one configuration object is not applied to other configuration objects of the array.
+
+```js
+// eslint.config.js
+
+export default [
+    { name: "config-1", files: "src/*", ignores: [".dist/*"], ... }, 
+    { name: "config-2", files: "scripts/*", ignores: [".others/*"], ... }, 
+    // ...
+];
```

Comment:
```suggestion
export default [
    { name: "config-1", files: "src/**/*.js", ignores: ["**/*.config.js"], ... }, 
    { name: "config-2", files: "scripts/**/*.js", ignores: ["**/dir/**"], ... }, 
    // ...
];
```

Here we can show that `config-1` is ignoring a file and `config-2` is ignoring files inside a directory.